### PR TITLE
rewrite the README

### PR DESCRIPTION
**Pull request by doudou**
Depends on:
- [x] #42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,29 +6,164 @@ workspace.
...
+
+The extension relies on the debugger provided by the Ruby extension. Some gems
+[need to be installed](#install) for this to be functional.
+
+The Ruby launch configurations are therefore the [Ruby debugger configurations](https://github.com/rubyide/vscode-ruby/wiki/2.-Launching-from-VS-Code). The
+same expansions that are available in C++ entries are also available in the Ruby
+entries, that is:
+
+- `${rock:srcDir}` expands to the package's source directory
+- `${rock:buildDir}` expands to the package's build directory
```
Comment (g-arjones) [inline README.md]: Isn't this undefined for ruby packages?

**Comment by g-arjones** [comment]
Cool!

File: README.md (hunk 1 above)
Comment (doudou) [reply README.md]: Good point ...

**Comment by doudou** [comment]
Fixed the mention of `buildDir` for ruby packages and merged.

**Pull request merged**